win32ss/user/ntuser/desktop.c

```diff
@@ -2957,26 +2958,36 @@ NtUserGetThreadDesktop(DWORD dwThreadId, DWORD Unknown1)
     }
 
     Status = PsLookupThreadByThreadId((HANDLE)(DWORD_PTR)dwThreadId, &Thread);
-    if (!NT_SUCCESS(Status))
+    if (!NT_SUCCESS(Status) || !Thread)
```

Comment:
PsLookupThreadByThreadId won't return success without also returning a valid thread pointer, so this addition is unnecessary.